nvflare/app_opt/xgboost/histogram_based_v2/fed_controller.py

```diff
@@ -50,8 +60,14 @@ def __init__(
         )
         self.int_client_grpc_options = int_client_grpc_options
         self.in_process = in_process
+        self.server_handler_id = server_handler_id
 
     def get_adaptor(self, fl_ctx: FLContext):
+        if not self.server_handler_id:
```

Comment:
What is this for? Should we always create the handler so the user will never need to create it?

Reply:
This is for future use. Removed.